packages/eslint-config-airbnb-base/rules/es6.js

```diff
@@ -164,6 +163,7 @@ module.exports = {
 
     // import sorting
     // https://eslint.org/docs/rules/sort-imports
+    // Defer to Jarrod's work on getting similar behavior to TS-Hero
```

Comment:
@MikeMatusz and Jarrod and Zach take this offline and come back with a recommendation in the next meeting.